pkg/util/util.go

```diff
@@ -17,7 +17,7 @@ func IsMountpoint(path string) bool {
 // Synchronously invoke a command, propagating stdout and stderr
 // to the current process's stdout and stderr
 func RunCmdSync(cmdName string, args ...string) error {
-	logrus.Debugf("Running: %s %s", cmdName, strings.Join(args, " "))
+	olog.Printf("Running: %s %s", cmdName, strings.Join(args, " "))
```

Comment:
This is optional log statement, it will only appear when `--verbose` is provided.